### DISCUSSION TITLE
vue 3.2.x supports

### DISCUSSION THREAD

**Discussion by xingxiuyi** [FranckFreiburger/vue3-sfc-loader#112] [Ideas]
any plan upgrade to vue / sfc to 3.2.x (latest is 3.2.19)?

after clone this repos and change package.json vue / sfc version. build has failed.

or provide esm version (small size) use externals dependencies, import vue / sfc from local , not only buildins?

**Comment by FranckFreiburger** [comment]
vue 3 @ 3.4.15 
see vue3-sfc-loader-node
